host/src/request.rs

```diff
@@ -195,15 +195,20 @@ pub struct ProofRequestOpt {
     /// The proof type.
     pub proof_type: Option<String>,
     #[command(flatten)]
+    #[serde(flatten)]
```

Comment:
what's the diff between #[command(flatten)] & #[serde(flatten)]?

Reply:
`#[command(flatten)]` only handles the flattening for CLI passed params, while `#[serde(flatten)]` handles it for file and request params